test/everypolitician_test.rb

```diff
@@ -100,7 +100,7 @@ def test_alternative_constant_name
   end
 
   def test_retrieving_popolo
-    VCR.use_cassette('popolo') do
+    VCR.use_cassette('popolo', :record => :new_episodes) do
```

Comment:
Great catch! I usually write these things as `record: :new_episodes`, so shame on me for copy-pasting from the webpage.